docs/kratos/further-reading/kratos-video-tutorials.mdx

```diff
@@ -1,119 +0,0 @@
----
-id: kratos-video-tutorials
-title: Video Tutorials
```

Comment:
@vinckr do you agree to remove this and if so do we need this document somewhere else?

Reply:
Just my two cents:

https://www.ory.sh/docs/kratos/further-reading/kratos-video-tutorials
This page hosts 2 videos. One is also in the quickstart section for Kratos so we're not losing it. The other one isn't used anywhere else so it's a goner. Both of these are very old so I don't think that scrapping them does us any harm.